app/src/main/java/com/framgia/moviedb_31/view/homescreen/PagerAdapter.java

```diff
@@ -0,0 +1,48 @@
+package com.framgia.moviedb_31.view.homescreen;
+
+import android.content.Context;
+import android.support.annotation.NonNull;
+import android.view.View;
+import android.view.ViewGroup;
+import android.widget.ImageView;
+import com.bumptech.glide.Glide;
+import com.bumptech.glide.request.RequestOptions;
+import java.util.List;
+
+public class PagerAdapter extends android.support.v4.view.PagerAdapter {
```

Comment:
em nên import android.support.v4.view.PagerAdapter
```suggestion
public class PagerAdapter extends PagerAdapter {
```